nepal_compliance/overrides/salary_slip.py

```
@@ -0,0 +1,175 @@
+    def submit_salary_slips_for_employees(self, salary_slips, publish_progress=True):
+        submitted = []
+        unsubmitted = []
+        frappe.flags.via_payroll_entry = True
+        count = 0
+
+        for entry in salary_slips:
+            try:
+                salary_slip = frappe.get_doc("Salary Slip", entry[0])
+                salary_slip.submit()
+                submitted.append(salary_slip)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)